modules/new_serverpod_auth/serverpod_auth_core/serverpod_auth_core_server/test/session/integration/auth_sessions_admin_test.dart

```diff
@@ -142,6 +142,139 @@ void main() {
     );
   });
 
+  withServerpod('Given an auth session,',
+      (final sessionBuilder, final endpoints) {
+    late Session session;
+    late UuidValue authUserId;
+    late UuidValue authSessionId;
+
+    setUp(() async {
+      session = sessionBuilder.build();
+
+      authUserId = await createAuthUser(session);
+
+      // ignore: unused_result
+      await authSessions.createSession(
+        session,
+        authUserId: authUserId,
+        scopes: {},
+        method: 'test',
+      );
+
+      authSessionId = (await AuthSession.db.find(session)).single.id!;
+    });
+
+    test(
+      'when calling `deleteSessions`, then it is deleted and tuple returned.',
+      () async {
+        final deleted = await AuthSessionsAdmin().deleteSessions(session);
+
+        expect(deleted, hasLength(1));
+        expect(deleted.single.$1, authUserId);
```

Comment:
Avoid using unnamed records on public interfaces to avoid using `$n` access, which is error prone under refactorings. In particular when `$1` and `$2` are the same type, but not interchangeable like here.